spec/factories/plans.rb

```diff
@@ -80,5 +80,38 @@
                     user: evaluator.user)
       end
     end
+    trait :completed do
+      transient do
+        # Often we need to test a completed degree for its last quarter, which
+        # is the quarter in which the last courses needed to complete the
+        # degree were taken. This allows for specifying the last quarter from
+        # the test. All other courses will be taken in a previous quarter.
+        #
+        # Note: can take either a 6-digit quarter code or a Quarter object
+        last_quarter { Quarter.current_quarter.previous_quarter }
+      end
+
+      after(:create) do |plan, evaluator|
```

Comment:
Unused block argument - `plan`. If it's necessary, use `_` or `_plan` as an argument name to indicate that it won't be used.